packages/navigation/src/hooks/useWebviewNavigate.ts

```diff
@@ -105,6 +105,9 @@ function getMinimalAction(
   while (
     currentAction.payload &&
     'name' in currentAction.payload &&
+    currentAction.payload.name &&
+    'params' in currentAction.payload &&
+    currentAction.payload.params.screen &&
```

Comment:
This fixes issue where the navigating to the same named screen in a stack would dispatch an empty navigation payload (`{}`) as all screens were equal and thus removed.